spec/support/govuk_i18n_cov.rb

```diff
@@ -0,0 +1,81 @@
+# frozen_string_literal: true
+
+class GOVUKI18nCov
```

Comment:
It'd be more conventional for this to be capitalised as Govuk. Though I wonder if it needs a Govuk prefix as it's not specific to us.

Reply:
Fair enough - I was being cautious :-).